### PR TITLE
Satisfaction 75 turn off oled on suspend

### PR DESCRIPTION
**Pull request by MarMed**
a0089aa34519ec4fcf7cee6a352424e4b7cba7da makes changes to the OLED, I believe this causes the OLED to freeze and stay on after host suspend, I included the snippet of code that sleeps the OLED within suspend_power_down_user to fix this issue. This may not be the best way, but in my testing it works, open to improvements.

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/cannonkeys/satisfaction75/led.c [hunk 1]
```diff
@@ -19,6 +19,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "led_custom.h"
 #include "satisfaction75.h"
 
+#ifdef QWIIC_MICRO_OLED_ENABLE
+#include "micro_oled.h"
```
Comment (drashna) [inline keyboards/cannonkeys/satisfaction75/led.c:23]: "no such file exists", in travis CI.

**Comment by tzarc** [comment]
@awkannan can you confirm you're okay with this patch?

File: keyboards/cannonkeys/satisfaction75/led.c (hunk 1 above)
Comment (fauxpark) [reply keyboards/cannonkeys/satisfaction75/led.c:23]: only for the `tester` keymap, which has `QWIIC_ENABLE = no`... but that should result in `QWIIC_MICRO_OLED_ENABLE` not being defined: https://github.com/qmk/qmk_firmware/blob/master/drivers/qwiic/qwiic.mk

**Comment by MarMed** [comment]
Is there anything I can do to help get this PR reviewed?

**Comment by fauxpark** [comment]
Potentially obsoleted by https://github.com/qmk/qmk_firmware/pull/14174

**Comment by tzarc** [comment]
> Potentially obsoleted by #14174

Definitely obsoleted. Next merge from `develop` will swap Satisfaction75 to the normal OLED driver, which has timeout already present.

Closing.

**Pull request closed without merging**